internal/runbits/dependencies/changesummary_test.go

```diff
@@ -1,168 +0,0 @@
-package dependencies
```

Comment:
I deleted it because it would take a lot of effort to update because of all the mocking, and because this was effectively an integration test masquerading as a unit test.

I'll file a new ticket for us to add integration tests.

Reply:
DX-2789